nemo/collections/asr/parts/dataset.py

```diff
@@ -376,14 +426,17 @@ def __init__(
         self.trim = trim
         self.load_audio = load_audio
 
-        self.labels = labels
-        self.num_commands = len(labels)
+        self.labels = labels if labels else self.collection.uniq_labels
```

Comment:
If labels are now optional, can you reflect that in the init() and docstring?

Reply:
Done